docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -33,7 +33,49 @@ Keeping informed
 To track changes that affect collections:
 
 * Join the `Collection Maintainers & Contributors forum group <https://forum.ansible.com/g/CollectionMaintainer>`_.
-* Subscribe to the `Bullhorn <https://forum.ansible.com/c/news/bullhorn/17>`_ Ansible contributor newsletter.
+* Subscribe to the :ref:`bullhorn` Ansible contributor newsletter.
+
+.. _coll_wg_reqs:
+
+Communication and Working Groups
+================================
+
+Forum overview
+--------------
+
+The :ref:`ansible_forum` is our default communication platform.
+
+In the context of organizing communication around Ansible collections, you need to understand the following notions:
+
+* `Tags <https://forum.ansible.com/tags>`_: together with categories, the tags is the main feature used in the Forum to organize conversations around specific kinds of topics. Most of projects in the Forum have associated tags. In case of Ansible collections, a tag name is usually based on technology the collection automates, for example, ``kubernetes``, ``windows``, or ``postgresql``.
+* `Forum groups <https://forum.ansible.com/g>`_: allow you to organize users, manage permissions, have a working group page that provides all related information, manage notifications including automatic tag subscription for members, mention or message the whole group, and more. An example of collection working group is the `PostgreSQL Ansible Collection Working Group <https://forum.ansible.com/g/PostgreSQLTeam>`_.
+
```

Comment:
Is it worth adding here that only request a forum group if you have a dedicated team that will interact with the forum for this collection. 

We don't want a bunch of groups added that don't add value over just the collection maintainer watching the tag...

Reply:
@samccann i think it's worth adding anyway. In many cases, people will join afterwards as long as there's invitation in README and just presence on the forum. Take a look at, say, postgresql, mysql, clickhouse, execution environments teams: there were 2-3 folks originally, now there are at least x3 more.
sure, there can be cases like you mentioned and we could purge them from time to time if needed.
joining a group gives a sense of belonging to a sub-community compared to just watching a tag and it doesn't cost us much.